lib/cubit/lnurl/lnurl_cubit.dart

```diff
@@ -12,7 +15,19 @@ class LnUrlCubit extends Cubit<LnUrlState> {
   final _log = Logger("LnUrlCubit");
   final BreezSDKLiquid _liquidSdk;
 
-  LnUrlCubit(this._liquidSdk) : super(LnUrlState.initial());
+  LnUrlCubit(this._liquidSdk) : super(LnUrlState.initial()) {
+    _initializeLnUrlCubit();
+  }
+
+  void _initializeLnUrlCubit() {
+    late final StreamSubscription streamSubscription;
+    streamSubscription = _liquidSdk.walletInfoStream.listen(
+      (walletInfo) {
+        fetchLightningLimits();
```

Comment:
The fetchLightningLimits is in LnurlCubit. Do we use the limits also on regular lightning payments?

Reply:
> The fetchLightningLimits is in LnurlCubit. Do we use the limits also on regular lightning payments?

Yes, we should. It'll be addressed in scope of another PR.